### DISCUSSION TITLE
Is there a way to force object detection on camera feed (i.e. not triggered by movement)

### DISCUSSION THREAD

**Discussion by cat101** [blakeblackshear/frigate#8899] [General]
I'm thinking about this in the context of presence detection for home automation. I know that Frigate tracks objects, but eventually it will stop tracking (which is correct). 

If I could trigger object detection at will, I could decide whether to turn light off if nobody is on the room

Thanks!

**Comment by NickM-27** [comment]
Not sure what you mean by frigate will stop. Frigate tracks objects as long as one is visible. Even when objects are stationary they are tracked.

**Comment by cat101** [comment]
Mmm maybe I was misled by the Events UI where objects stop being "in progress" after a while. Let me rephrase the question: Is there a way (i.e. API) to get a list of all the object being tracked (including all the stationary ones)? I have tried with `/api/events` but all I see is the same events I would see on the events UI.

Am I missing something?? Should I change settings so that objects can remain in progress for ever?  (in order to be able to get a list of people in a room)

Thanks @NickM-27 !

**Reply by NickM-27** [reply]
If an object was lost and stops being tracked that's either a config issue or the object was lost due to detection. 

No, there is no raw API to get tracked objects

**Reply by cat101** [reply]
Thanks @NickM-27 for your guidance.... Is this correct:

1. `In progress` events will remain as such if the object remains there. For example: if I put a bike in front of a camera for a day, the event will be in progress for 1 day. Naturally, lighting conditions and noise may end the event and re-start it.
2. A sub set of the `in progress` events will have `active objects`. Frigate will only record `active objects` by default

Cheers

PS:  Should I mark this conversation a resolved or just leave open?? (what it the community practice here??)

**Reply by NickM-27** [reply]
1. Yes, that is the goal. Frigate 0.13 has had many improvements in this area and there are more planned in the future. 
2. The active objects recordings mode refers to any object being active.

**Reply by NickM-27** [reply]
You can close as resolved if you like, for discussions it isn't a big deal either way
